core/src/avm2/object/xml_list_object.rs

```diff
@@ -766,20 +766,14 @@ impl<'gc> TObject<'gc> for XmlListObject<'gc> {
                                 .xml_object_child(0, activation)
                                 .expect("List length was just verified");
 
-                            if matches!(
```

Comment:
According to avmplus this should not match CData? https://github.com/adobe/avmplus/blob/858d034a3bd3a54d9b70909386435cf4aec81d21/core/XMLListObject.cpp#L365

Reply:
Seems you are right. I guess we need to replicate the bug in avmplus.

Reply:
I added a note.